pkg/plugins/client_test.go

```diff
@@ -254,9 +255,12 @@ func TestClientWithRequestTimeout(t *testing.T) {
 
 	switch e := err.(type) {
 	case *url.Error:
-		err = e.Err
+		assert.Assert(t, e.Timeout())
```

Comment:
Why not do something like, define the error interface locally:

```go
type timeoutError interface {
  Timeout() bool
}
```

And type check against that?